planetiler-core/src/main/java/com/onthegomap/planetiler/reader/ShapefileReader.java

```diff
@@ -125,7 +126,9 @@ private ShapefileDataStore open(Path path) {
       } else {
         throw new IllegalArgumentException("Invalid shapefile input: " + path + " must be zip or shp");
       }
-      return new ShapefileDataStore(uri.toURL());
+      var store = new ShapefileDataStore(uri.toURL());
+      store.setCharset(Charset.forName("UTF8"));
```

Comment:
Thanks for bringing this up! I noticed the same issue when trying to read the natural earth shapefile.  One concern here is that standard encoding for shapefiles is technically [ISO8859-1](http://en.wikipedia.org/wiki/ISO/IEC_8859-1) so I'd hesitate to always use UTF-8.

It looks like shapefiles that have a different encoding actually contain a ".cpg" file next to the .shp file with the text of the shapefile though, so maybe we could use that if it exists?

Reply:
Interesting point with the `.cpg`. QGIS seems to get the encoding right without further configuration: 
![image](https://user-images.githubusercontent.com/53421382/158561403-9724ece3-8e7a-410c-a15d-e4cb4502ecf1.png)

Reply:
Hmm you're right.. the shapefile reader should be doing this automatically.  From the [geotools release page](https://docs.geotools.org/stable/userguide/welcome/upgrade.html) it looks like starting with version 26 (we're on 26.3) they attempt to use cpg file to determine the encoding. I wonder why that's not working here?

Reply:
https://docs.geotools.org/stable/javadocs/org/geotools/data/shapefile/ShapefileDataStore.html#isTryCPGFile-- returns `false`, i.e., by default it does not access the `.cpg` file. Let's check if I can change the behavior with `setTryCPGFile()`...

Reply:
[2d29b3e](https://github.com/onthegomap/planetiler/pull/137/commits/2d29b3e94154695d6c04be67ec992e7d6fbb6a7f)